pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/new/index.md

```diff
@@ -53,10 +53,25 @@ enterprise: false
 
     This will create a unique name every time you run it, so use it with forethought.
 
+1.  (Optional) To create a cluster using a custom AMI built using [KIB][KIB], first that image must have been built in KIB.  Then perform the export and name the custom AMI. Set the environment variable for the AMI you choose: 
+
+```bash
+export AWS_AMI_ID=ami-<ami-id-here>
+```
+
+After export, run the following command:
+
+```bash
+dkp create cluster aws --cluster-name=${CLUSTER_NAME} \
+--ami=${AWS_AMI_ID} \
+--dry-run \
+--output=yaml \
+> ${CLUSTER_NAME}.yaml
+```
 
 ## Create a new AWS Kubernetes cluster
 
-1.  Ensure your AWS credentials are up to date. Refresh the credentials:
+1.  Ensure your AWS credentials are up to date. Use step 1 command to refresh the credentials if you are using Static Credentials, otherwise proceed to Step 2:
```

Comment:
```suggestion
1.  Ensure your AWS credentials are up to date. If you are using Static Credentials use the following command to refresh the credentials. Otherwise, proceed to Step 2:
```